cli-hangman.js

```diff
@@ -0,0 +1,96 @@
+var inquirerUserPrompt = require("inquirer");
+var Word = require("./word.js");
+
+var tries = 0;
+var lettersArray = [];
+var currentString = '';
+var currentWord;
+var lettersGuessed = [];
+
+Word.prototype.rightGuesses = 0;
+
+startGame = function() {
```

Comment:
Are these methods? Variables? Functions? They are defaulting to variables here but it is more concise to use the correct identifier for your future reference and anyone that will see this for the first time.